data/core/units/monsters/Fungoid_Body.cfg

```diff
@@ -0,0 +1,275 @@
+#textdomain wesnoth-units
+
+[unit_type]
+    id=Fungoid Body
+    name= _ "Fungoid Body"
+    race=monster
+    image=units/monsters/core-compact.png
+    image_icon="units/monsters/core-compact.png~SCALE(72,72)"
+    [standing_anim]
+        [frame]
+            duration=100
+            image="units/monsters/core-compact.png"
+            auto_hflip=no
+        [/frame]
+    [/standing_anim]
+    [resistance]
+        blade=110
+        pierce=110
+        impact=80
+        fire=150
+        cold=150
+        arcane=200
+    [/resistance]
+    hitpoints=180
+    movement_type=smallfoot
+    movement=0
+    experience=150
+    {AMLA_DEFAULT}
+    level=3
+    alignment=chaotic
+    advances_to=null
+    cost=403
+    usage=mixed fighter
+    description= _ "This thing is impossible to describe, no one has seen anything like it before."
+    {DEFENSE_ANIM_FILTERED ("units/monsters/core-compact.png") ("units/monsters/core-compact.png") ("wail.wav") (direction=n,ne,se,s)}
+    {DEFENSE_ANIM_FILTERED ("units/monsters/core-compact.png~FL(horiz)") ("units/monsters/core-compact.png~FL(horiz)") ("wail.wav") (direction=nw,sw)}
+
+    [abilities]
+        {ABILITY_REGENERATES}
+        [dummy]
+            id=miasma
+            name= _ "miasma"
+            description=_ "This unit passively harms those not of its side at a distance and rate equal to its level."
```

Comment:
I think this should probably have a special note as well.

I also think there should be no need to use an event for that. We can already easily make a "harms" ability… as far as I can tell the only issue is that it can't specify a damage type.

Reply:
A special note makes sense, agreed.

The cold damage type is preferred. Though maybe at this low of a damage level precision issues make damage type modifiers either ineffectual or a bit too strong. I will have to test.

Reply:
That note on harms and damage types was sort of wishful thinking for a feature we don't have, more than a request for a change.